Soundex.Tests.cpp

```diff
@@ -5,5 +5,34 @@ TEST(SoudexTestsuite, ReplacesConsonantsWithAppropriateDigits) {
  //AAA
   char soundex[5];
   generateSoundex("AX", soundex);
- // ASSERT_EQ(soundex,"A200");
+  //ASSERT_EQ(soundex,"A200");
 }
+
+TEST(SoudexTestsuite1, ReplacesConsonantsWithAppropriateDigits1)
```

Comment:
why multiple test suites? you can group  multiple test cases  in one Test Suite
SoudexTestsuite1,SoudexTestsuite2,SoudexTestsuite3